tests/bdd/steps/feature_steps/fixture.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Fixture helpers
+  Scenario: the user is a {role}
+    Given I load the fixture '{role}.ini' as 'role'
+      And I update the fixture 'params' from fixture 'role'
+
+  Scenario: the user is an {role}
+    Given the user is a {role}
+
+  Scenario: the request is for resource '{resource_id}'
```

Comment:
_resource_ is a slightly vague term. Not that I have a better suggestion...

Reply:
The LTI spec is full of hideously vague terms, but I'm afraid that's what they call an assigment _some_ of the time. In this case it's something linked to via a `resource_link_id`. I honestly don't know if it's a `resource_link` or a `resource`. Both are super vague.